analyzer/src/main/kotlin/Analyzer.kt

```diff
@@ -346,7 +346,7 @@ private fun resolveConfiguration(
 
     curationProviders.forEach { (id, curationProvider) ->
         val (curations, duration) = measureTimedValue {
-            curationProvider.getCurationsFor(packageIds).values.flatten()
+            curationProvider.getCurationsFor(packageIds).values.flatten().distinct()
```

Comment:
Commit message: I do not understand how the fact that "a single curation may apply to multiple packages" is a precondition for having duplicates (as that's how the sentence sounds to me). Even without applying to multiple packages, i.e with hard-coded ids without ranges, two different files in a curations directory could be copies of each other, resulting in a duplicate.

So maybe just say: For curations for a given id from a single provider no ordering is assumed. As such, having duplicates (like by accidental copies of files in a `FilePackageCurationProvider`) does not make any sense. So remove any duplicates to be on the safe side.